app/Console/Command/GeocodePlaceCommand.php

```diff
@@ -38,7 +38,7 @@ protected function dispatchGeocodingCommand(string $placeId, OutputInterface $ou
 
         if (!isset($jsonLd['address'][$addressLanguage])) {
             // Some places have an address in another language then the main language or `nl`
-            $addressLanguage = array_key_first($jsonLd['address']);
+            $addressLanguage = array_key_first($jsonLd['address'] ?: []);
```

Comment:
Not sure if this will fix this error https://publiq-vzw.sentry.io/issues/5027675756/?project=5435528&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=30d&stream_index=3

Reply:
You are right. It solved the fatal error, but there is still a warning. The check with ?? is better here, that supresses both errors.

Reply:
For me it feels more logical to add an early return for when the `address` property is missing. When using the query it is not possible to have a missing address, now places can have a missing address.

```
if (!isset($jsonLd['address'])) {
 // Log missing address
 // Do early return
}
```